bindings/wasm/test/examples/import-manifold.ts

```diff
@@ -0,0 +1,68 @@
+// This example shows how to import a 3d model and
+// manipulate it within manifoldCAD.
+//
+// There are two functions that import models with different purposes.
+// `importModel()` will import a model for visualization purposes.
+//
+// `importManifold()`, as shown here, will make a best-effort at converting
+// the source model into a fully manifold object that can be manipulated
+// or combined like any other manifoldCAD object.
+
+import {importManifold, Manifold} from 'manifold-3d/manifoldCAD';
+const {compose} = Manifold;
+
+const csg = async () => {
+  // glTF (.glb or .gltf) models have a defined scale of 1 unit to 1 metre.  On
+  // the other hand, manifoldCAD has a defined scale of 1 unit to 1 millimetre.
+  // These models are 100m and 120m wide, respectively.  Smaller than you'd
+  // expect for a moon, but larger than you'd expect for a model.
```

Comment:
Maybe we should just resave them at 0.1m scale so we can skip the note...

Reply:
Right, definitely avoid implicit scaling - I just figured the example might be simpler if the GLBs are already at a reasonable scale. Either way is fine though.